src/compiler/scala/tools/nsc/reporters/ConsoleReporter.scala

```diff
@@ -59,13 +59,13 @@ class ConsoleReporter(val settings: Settings, reader: BufferedReader, writer: Pr
 
   /** Prints the number of warnings and errors if there are any. */
   def printSummary(): Unit =
-    for (k <- List(WARNING, ERROR) if k.count > 0) printMessage(s"${countAs(k.count, label(k))} found")
+    for (k <- List(WARNING, ERROR) if k.count.get() > 0) printMessage(s"${countAs(k.count.get(), label(k))} found")
 
   def display(pos: Position, msg: String, severity: Severity): Unit = {
```

Comment:
will need some synchronization here - there are inherent races

It doesn't get hit that much though, so we should be just able to synchronize on the AbstractReporter methods I think